### DISCUSSION TITLE
Version 1.4.2.0-release `<Grand Adventure's End>

### DISCUSSION THREAD

**Discussion by zer0Kerbal** [zer0Kerbal/Taerobee#144] [Announcements]
[![Taerobee Version][MOD:shd:stat]][MOD:url] [![KSP version][KSP:shd:stat]][KSP:url]
[![License][LIC:shd]][LIC:url]  ![AVC .version files][AVCVAL:shd]

[![GitHub Pages][MOD:pages:shd]][MOD:pages]

---

# Version 1.4.2.0-release `<Grand Adventure's End>`

* 25 Mar 2022
* Release for Kerbal Space Program [KSP 1.12.x]

<h4 style="border:0.5px solid Tomato; background-color: #bada55; color: #FF0000; text-align:center">
<b>DO A CLEAN INSTALL:</br> DELETE EXISTING INSTALLATION THEN RE-INSTALL</b></h4>
<p style="border:0.5px solid Tomato; background-color: #bada55; color: #FF0000; text-align:center">Download from <a href="https://www.curseforge.com/kerbal/ksp-mods/Taerobee/files">CurseForge</a></p>

## Changes

* U

### Status

* Release changes (internal)
  * closes #112 -  Taerobee Release 1.4.2.0-release - Spanish Waterfall Snacks
  * closes #113 -  Version 1.4.2.0-release
  * closes #114 -  1.4.2.0 Verify Legal MumboJumbo
  * closes #115 -  1.4.2.0 Update documentation
  * closes #116 -  1.4.2.0 - Social Media
  * closes #110 -  Minor Error in the ReadMe.md
  * updates #118 -  make sure all tags are complete

---

### See More

* Changelog Summary for more details of changes : See [ChangeLog][MOD:chlog]  
* Discussions and news on this mod : See [Discussions][MOD:discu] or [KSP Forums][MOD:url]  
* Known Issues for more details of feature requests and known issues : See [Known Issues][MOD:known]  
* GitHub Pages : See [Pages][MOD:pages]

### Localization

>* ![English][EN] English
>* ![Mexican][ES] Spanish (Español) [incomplete]
>* ![Mexican][ME] Mexican Spanish (Español Mexicano) [incomplete]
>* ![Russian][RU] Russian (Русский)  [incomplete]
>* [ ]  ***your translation here***
>
> HELP WANTED - See the [README in the Localization folder][MOD:local] for instructions for adding or improving translations. [GitHub][GitHub:url] push is the best way to contribute. *Additions and corrections welcome!*

---

### How to support this and other great mods by [`zer0Kerbal`][zer0Kerbal]

[![Support][PAYPAL:img]][PAYPAL:url] [![Patreon][PATREON:img]][PATREON:url] [![Github Sponsor][GSPONS:img]][GSPONS:url] [![Buy zer0Kerbal a snack][BMCC:img]][BMCC:url]

---

<div style="border:0.5px solid Tomato; background-color: #BADA55; color: #FF0000; text-align:center">
<p><b>*red box below is a link to forum post on how to get support*</p>  
<a href = "https://forum.kerbalspaceprogram.com/index.php?/topic/83212-*">
  <p><img src = "https://i.postimg.cc/vHP6zmrw/image.png" alt="How to get support"></p></a>
</br>
<p>Be Kind: Lithobrake, not jakebrake! Keep your Module Manager up to date</p></div>

---

[MOD:url]: https://forum.kerbalspaceprogram.com/index.php?/topic/205846-* "Taerobee  on the forums"
[MOD:chlog]: https://raw.githubusercontent.com/zer0Kerbal/Taerobee/master/changelog.md "Changelog"
[MOD:discu]: https://github.com/zer0Kerbal/Taerobee/discussions "Discussions"
[MOD:issue]: https://github.com/zer0Kerbal/Taerobee/issues "Issue Tracker"
[MOD:known]: https://github.com/zer0Kerbal/Taerobee/wiki/Known-Issues "Known Issues"
[MOD:licns]: https://github.com/zer0Kerbal/Taerobee/blob/master/LICENSE "Repo License"
[MOD:local]:  https://github.com/zer0Kerbal/Taerobee/blob/master/GameData/Taerobee/Localization/readme.md "Localization"
[MOD:pages]: https://zer0kerbal.github.io/Taerobee/ "GitHub Pages"
[MOD:pages:shd]: https://github.com/zer0Kerbal/Taerobee/actions/workflows/pages/pages-build-deployment/badge.svg "GitHub IO"

[MOD:shd:stat]: https://img.shields.io/badge/Taerobee%20(TBEE)%20v-1.4.2.0--release-BADA55.svg?style=plastic&labelColor=darkgreen "1.4.2.0-release"

[CRSFG:url]: https://www.curseforge.com/kerbal/ksp-mods/Taerobee/files "Curseforge"
[GITHUB:url]: https://github.com/zer0Kerbal/Taerobee/ "GitHub"

[KSP:url]: http://kerbalspaceprogram.com/ "Kerbal Space Program"
[KSP:shd:stat]: https://img.shields.io/badge/KSP-1.12.3-blue.svg?style=plastic&logo=data:image/webp;base64,UklGRpAGAABXRUJQVlA4TIQGAAAvH8AHEE0obNsGDakwXkT/Q4chj76jn1yYjSRjH6H+6xQqRUkkSc5ckHD+NUUVhx4+RNu2bTTl/3OTVhD6H8u6fF8dDYm40CK7N0CjAxK52rYtjfQRAvGNjbu7V+vuLp2fwhzA6gG4li6de+nuO+4uJBlkwoTA//90kENgtZZs27ZpO2Oufa5vbNsq2Sw6qaa9r3n/kZJTex/wSrFt27o62kuSJNuqrdTa1+/7uLu7uzQZAk0mRN/mAz2CAbi723vvy5WztxxIAAiw+dh2bdvWZJuT12y1bdu2bdu2bbs3AQABlsED+y50lDqLE4pf/Uxe8KO77HFes9hvxu1p3O2Q09LVg0NVjo7Z5U6AvwBSOfivKdQTujOnXNG8hzNhqYSMMHJ+MwWWyTSjNUoP1jWHp1ZNmgB8TMH3Tl33mvoVb8uIj3umyIJLsAmYoCIqUCVMCN5WrW78Qi+AkeaGOkWFk3QbRmIWHpIZSURXDxhJVlJiYZ6kLEgeEjIRCBmxebhCTWlTJNaXLystGysVToPOAFJJjqSPNOXftSVbXPQjxrzVaFNzXvBonWJlonIVTJabAfjZ0Jxufl5GPntEHKjB7PIXtg6eYHywJn3hTdijjXUain94KjfCfg1hm³AZRR4N16VMXBrsDVYDCjOIMmGFsAab8rdY7+kIw0BgtRgL1DKJeuNZdQ9f5RAGGfOQfZxexj7t37RkjNp4rPDIHOJVyPBF+XHKsnQk4SWRBl8Wot74WtkBqidQggbyj2vW1WEFG6JfUZ8UxhVzOmAUdKTYZ3yxR2QwIIAQdA9YgiRBZWyCguiOER9y9IauiyGqR0EayVLiEVwmXmF5+Z2jfohj8i1q8ybRahCBAXZBCBUqLgAACQFzp5M/o5wzCDcCJZEOF9EzggM7NEd18c1Q85gPBldtg6mwB1N+hzyTSFp5jWM+WpCRBEBAjIaUxJI80o3s6vDYJ7gNXuAUfOacvsLvLhNIMaMwh8GccsXjtuiK+wkxZ/kVdOCzoCeAOn+N7H1rU1YT35MgAWY9yD5wMWI7j7G976bSh8LxDx2jI3IJRAnJ+RQQIU7gABnLd3vKSyMAd71mq7HgV3AJ+Sxh5Bz5O0z/gbTiNT8DxQssrrv7Uyx9Y7q6BQVUoIGonF/FfoC/RqwVHlviCRSABDvLaCUfZlz1cNENn6adNTz+AF8v8KWQAFL+QMkpCthFkpvvq7+2/to6AIBhyvPdrUr3g1XwBEJPMeP/98SxWWvs3nHIhgNnsY1RWVyw+Guh5IZLgj9WzPi/iEk9gPPGLAni3aRlsqlGQ9jP4z9Q/s/xi3dw486abm6Bqi2l3qu9e9tQsdVMdG4zbVYqp5wJcBSA2W6sNmkwxuNk+nw3r4MFi/4EkTAhiRFk70jkqNDiWbDK65OLIn0s0wD/ucAp3XwVSsSNBAqjFUIXwFE0MihZQ6utQYV2oR+tShO7Ad6bswY3xd7qe0VrVxK9ZjueQy4TPnF8MCQGjoxSugSFgFMV4LiVwgsn/i+gXZ2FzplAduxnn0/OlW0uqf+M64MRPqzwrl+PnpKQSaXS49Ui7n2/ctFtHtAtiprzxq6WniwESvW5yUG1Xx6/8Hx8NKELwKZK15pV/EvXvm9ZMlx0aKUO98iUPaWvst/n8ZbOmkvGhcOQRWf5zj9dk9cfpad5oHN3Rns/wsuvy2puxz1Ziu96Q7/SOWoROzvNzFo5Z5+1BDej3OjQ/XymEkW9jr0em5g5SdX8VC2gf9xJb/RWCC5bIKWDgWcYf+K9Kje3zbQBh/F448wMLoICeUyJ330nXlPmawiRT/sblG4vWrbErgQaMzYbZcwbhSaNrwH+Tqa04jqrd3JZTvwbFxHFSVMAv5UZdEq+tQUupcis/5+MZNsxk9b8TPa7cMqdzzrh9FtD5v+vPACvJy7nDT69IP/Yx6EywGdTFsD5iU7bqkovJogzTjQm³iFTyp4jV4bjVKdcnv5/JrhokmpnGAIA4D/AXYCfVgoBXnrDkCqqCHRG529HeYB51Jy1z6nlW/gnVmzyxmVHxnQrxXxelcI0yN85udPl+//t2rzKzA+oluPTNjp6qY1PVduFVdo8ya+8E6p8KOZR+bLj6Vju9oi5dar0erTS8Z1x3/IITU3vyDRLiZWBZVH6CbqURTeLptD3pEPIR4W4QlHfTnRJzZBRJ8MlI8LmmEXLAdAxsqIYbSDGTt65GfF0cUL6aQQ=&labelColor=black "Kerbal Space Program"
[LIC:url]: https://creativecommons.org/licenses/by-nd/3.0/ "CC BY-ND 3.0 Unported"
[LIC:shd]: https://img.shields.io/endpoint?url=https://raw.githubusercontent.com/zer0Kerbal/Taerobee/master/json/license.json "CC BY-ND 3.0 Unported"
[AVCVAL:shd]:    https://github.com/zer0Kerbal/Taerobee/workflows/Validate%20AVC%20.version%20files/badge.svg?style=plastic&labelColor=black  
[zer0Kerbal]: https://forum.kerbalspaceprogram.com/index.php?/profile/190933-zer0kerbal/ "zer0Kerbal"
[PAYPAL:img]: https://img.shields.io/badge/Buy%20me%20some%20-LFO-00457C?style=for-the-badge&logo=paypal&labelColor=FFDD00 "PayPal"
[PAYPAL:url]: https://www.paypal.com/donate?hosted_button_id=DC22YHMEJREKL "PayPal"
[PATREON:img]: https://img.shields.io/badge/Patreon%20-be%20a%20Patron-FF424D?style=for-the-badge&logo=patreon "Patreon"
[PATREON:url]: https://www.patreon.com/bePatron?u=23390503 "Patreon"
[GSPONS:img]: https://img.shields.io/badge/Github%20-Sponsor-EA4AAA?style=for-the-badge&logo=githubsponsors "Github Sponsors"
[GSPONS:url]: https://github.com/sponsors/zer0Kerbal "Github Sponsors"
[BMCC:img]: https://img.shields.io/badge/Buy%20Me%20a%20-Snack!-FFDD00?style=for-the-badge&logo=buymeacoffee "Buy Me A Snack"
[BMCC:url]: https://buymeacoffee.com/zer0Kerbal "Buy Me A Snack"
[EN]: https://raw.githubusercontent.com/zer0Kerbal/zer0Kerbal/zed'K/Localization/img/American-flag-sm.png "American English"
[BR]: https://raw.githubusercontent.com/zer0Kerbal/zer0Kerbal/zed'K/Localization/img/Brazilian-flag-sm.png "Brasil"
[CN]: https://raw.githubusercontent.com/zer0Kerbal/zer0Kerbal/zed'K/Localization/img/Chinese-flag-sm.png "中文"
[DE]: https://raw.githubusercontent.com/zer0Kerbal/zer0Kerbal/zed'K/Localization/img/German-flag-sm.png "Deutsch"
[ES]: https://raw.githubusercontent.com/zer0Kerbal/zer0Kerbal/zed'K/Localization/img/Spanish-flag-sm.png "Español"
[FR]: https://raw.githubusercontent.com/zer0Kerbal/zer0Kerbal/zed'K/Localization/img/French-flag-sm.png "Français"
[IT]: https://raw.githubusercontent.com/zer0Kerbal/zer0Kerbal/zed'K/Localization/img/Italian-flag-sm.png "Italiano"
[JA]: https://raw.githubusercontent.com/zer0Kerbal/zer0Kerbal/zed'K/Localization/img/Japanese-flag-sm.png "日本語"
[KO]: https://raw.githubusercontent.com/zer0Kerbal/zer0Kerbal/zed'K/Localization/img/South-Korean-flag-sm.png "한국어"
[ME]: https://raw.githubusercontent.com/zer0Kerbal/zer0Kerbal/zed'K/Localization/img/Mexican-flag-sm.png "Español Mexicano"
[NL]: https://raw.githubusercontent.com/zer0Kerbal/zer0Kerbal/zed'K/Localization/img/Dutch-flag-sm.png "Dutch"
[NO]: https://raw.githubusercontent.com/zer0Kerbal/zer0Kerbal/zed'K/Localization/img/Norwegian-flag-sm.png "Norsk"
[PO]: https://raw.githubusercontent.com/zer0Kerbal/zer0Kerbal/zed'K/Localization/img/Polish-flag-sm.png "Polski"
[RU]: https://raw.githubusercontent.com/zer0Kerbal/zer0Kerbal/zed'K/Localization/img/Russian-flag-sm.png "Русский"
[SW]: https://raw.githubusercontent.com/zer0Kerbal/zer0Kerbal/zed'K/Localization/img/Swedish-flag-sm.png "Svenska"
[TW]: https://raw.githubusercontent.com/zer0Kerbal/zer0Kerbal/zed'K/Localization/img/Taiwanese-flag-sm.png "国语"
[HA]: https://raw.githubusercontent.com/zer0Kerbal/zer0Kerbal/zed'K/Localization/img/Hawaii-flag-sm.png "ʻŌlelo Pākē"

## What's Changed
* Update EnginesPlumeSWE.cfg by @AtomikkuSan in https://github.com/zer0Kerbal/Taerobee/pull/127
* Create es-es.cfg by @AtomikkuSan in https://github.com/zer0Kerbal/Taerobee/pull/126
* Create SnacksAT.cfg by @AtomikkuSan in https://github.com/zer0Kerbal/Taerobee/pull/125
* Create USI-LS.cfg by @AtomikkuSan in https://github.com/zer0Kerbal/Taerobee/pull/124
* Update readme.md by @AtomikkuSan in https://github.com/zer0Kerbal/Taerobee/pull/123
* Version 1.4.1.0 - Spanish Waterfall Snacks by @zer0Kerbal in https://github.com/zer0Kerbal/Taerobee/pull/130
* Update EnginesPlumeSWE.cfg by @AtomikkuSan in https://github.com/zer0Kerbal/Taerobee/pull/133
* Update en-us.cfg by @AtomikkuSan in https://github.com/zer0Kerbal/Taerobee/pull/141
* Create SlimJim.cfg by @AtomikkuSan in https://github.com/zer0Kerbal/Taerobee/pull/140
* 1.4.2.0-release <Grand Adventure's End> by @zer0Kerbal in https://github.com/zer0Kerbal/Taerobee/pull/143

## New Contributors
* @AtomikkuSan made their first contribution in https://github.com/zer0Kerbal/Taerobee/pull/127

**Full Changelog**: https://github.com/zer0Kerbal/Taerobee/compare/1.4.0.0-release...1.4.2.0-release

<hr /><em>This discussion was created from the release <a href='https://github.com/zer0Kerbal/Taerobee/releases/tag/1.4.2.0-release'>Version 1.4.2.0-release `<Grand Adventure's End></a>.</em>

**Comment by AtomikkuSan** [comment]
Finally, after all this time, it's done!

**Reply by zer0Kerbal** [reply]
agree. almost. one more release. I will feed Watson the en file in a week for the missing localizations, plus there is the three new parts to do volume math on, and to add drag_cubes to the new engine.
Then - it should be done. Until somebody else adds another compatibility patch. 📦
